### PR TITLE
fix forceOwnerCrown Plugin Spamming Errors in Console

### PR DESCRIPTION
**Pull request by nmsturcke**
Fix the plugin spamming errors in console when checking group chats, instead, log them as debugging for people to see if they want to enable verbose console logs.

image

### CODE REVIEW

**Comment by D3SOX** [comment]
I'd rather add a check if it's a guild than just adjusting the log level

**Comment by Vendicated** [comment]
"no guildid or userid" yet userid is defined? this seems more like a bug than having to adjust log level

**Comment by Vendicated** [comment]
uhh wrong button

**Comment by D3SOX** [comment]
I've looked into this and it seems like it's using the same method for both cases. I'd just remove the else parte as I haven't seen any other good indicator to differentiate between groups and guilds here.

**Comment by nmsturcke** [comment]
Another option is if the guild isn't in the GuildStore just ignore

**Comment by D3SOX** [comment]
wdym? Checking the GuildStore wouldn't help as it we need to decide upfront from where the method got triggered.

**Comment by nmsturcke** [comment]
@D3SOX

File: src/plugins/forceOwnerCrown.ts [hunk 1]
```diff
@@ -38,6 +38,11 @@ export default definePlugin({
         },
     ],
     isGuildOwner(props) {
+        // Check if channel is a Group DM, if so return false
+        if (props.channel.type === 3) {
```
Comment (D3SOX) [inline src/plugins/forceOwnerCrown.ts:42]:
```suggestion
        if (props?.channel?.type === 3) {
```

I've also thought about checking the type. Are you sure `3` is only used for group channels?

Comment (nmsturcke) [reply src/plugins/forceOwnerCrown.ts:42]: image
According to docs yes

**Review by D3SOX — APPROVED** [review]
LGTM now

**Review by Vendicated — APPROVED** [review]

**Pull request merged**